### DISCUSSION TITLE
Fuzzy search at workspace/root dir level

### DISCUSSION THREAD

**Discussion by sabitm** [LazyVim/LazyVim#1902] [Q&A] [answered]
Is there exists fuzzy search at workspace level on lazyvim already? It's like the workpsace version of `<space>sb` or `current_buffer_fuzzy_find`. I could not find one. There is also grep on root dir `<space>sg`, but I'm looking for the fuzzy one. TIA

**Comment by vic-stan** [comment]
hello! did you find anything relevant?

**Reply by sabitm** [reply]
See the selected answer

**Comment by frostified-io** [comment, marked as answer]
Is the LazyExtra _editor.fzf_ which effectively replaces search in `leader -> sg` and then first pressing `ctrl-g` to enable fuzzy search what you want?

Testing it briefly it seems to search recursively the folder I am working in in a fuzzy manner.

**Reply by sabitm** [reply]
Oh yes, thank you. Maybe this is the way. For everyone wondering the toggle to fuzzy search (from initial live grep) in `sg`, it is `ctrl+g`

**Reply by frostified-io** [reply]
Yes, I wrote ctrl+g inside < and > brackets, but it somehow got eaten up, even though the preview looked good. I corrected it for it to be less confusing. Glad we could solve this. 👍
